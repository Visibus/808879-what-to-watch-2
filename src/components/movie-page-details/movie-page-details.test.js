import MoviePageDetails from './movie-page-details';
import films from '../../mocks/films';
import {BrowserRouter} from 'react-router-dom';

it(`renders correctly`, () => {
  const tree = window.renderer.create(
      <BrowserRouter><MoviePageDetails
        film={films[0]}
        moreLikeThisFilms={films}
        isAuthorizationRequired={false}
        userData={{}}
        onOpenCloseFilm={jest.fn()}
      />
      </BrowserRouter>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
