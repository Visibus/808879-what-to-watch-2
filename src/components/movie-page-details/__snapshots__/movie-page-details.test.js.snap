// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <section
    className="movie-card movie-card--full"
    style={
      Object {
        "backgroundColor": "#C6CADF",
      }
    }
  >
    <div
      className="movie-card__hero"
    >
      <div
        className="movie-card__bg"
      >
        <img
          alt="Seven Years in Tibet"
          src="https://es31-server.appspot.com/wtw/static/film/background/Seven_Years_in_Tibet.jpg"
        />
      </div>
      <h1
        className="visually-hidden"
      >
        WTW
      </h1>
      <header
        className="page-header movie-card__head"
      >
        <div
          className="logo"
        >
          <a
            className="logo__link"
            href="/"
            onClick={[Function]}
          >
            <span
              className="logo__letter logo__letter--1"
            >
              W
            </span>
            <span
              className="logo__letter logo__letter--2"
            >
              T
            </span>
            <span
              className="logo__letter logo__letter--3"
            >
              W
            </span>
          </a>
        </div>
        <div
          className="user-block"
        >
          <a
            href="/mylist"
            onClick={[Function]}
          >
            <div
              className="user-block__avatar"
            >
              <img
                alt="User avatar"
                height="63"
                src="https://htmlacademy-react-2.appspot.comundefined"
                width="63"
              />
            </div>
          </a>
        </div>
      </header>
      <div
        className="movie-card__wrap"
      >
        <div
          className="movie-card__desc"
        >
          <h2
            className="movie-card__title"
          >
            Seven Years in Tibet
          </h2>
          <p
            className="movie-card__meta"
          >
            <span
              className="movie-card__genre"
            >
              Adventure
            </span>
            <span
              className="movie-card__year"
            >
              1997
            </span>
          </p>
          <div
            className="movie-card__buttons"
          >
            <button
              className="btn btn--play movie-card__button"
              onClick={[Function]}
              type="button"
            >
              <svg
                height="19"
                viewBox="0 0 19 19"
                width="19"
              >
                <use
                  xlinkHref="#play-s"
                />
              </svg>
              <span>
                Play
              </span>
            </button>
            <button
              className="btn btn--list movie-card__button"
              onClick={[Function]}
              type="button"
            >
              <svg
                height="20"
                viewBox="0 0 19 20"
                width="19"
              >
                <use
                  xlinkHref="#add"
                />
              </svg>
              <span>
                My list
              </span>
            </button>
            <a
              className="btn movie-card__button"
              href="/film/1/review"
              onClick={[Function]}
            >
              Add review
            </a>
          </div>
        </div>
      </div>
    </div>
    <div
      className="movie-card__wrap movie-card__translate-top"
    >
      <div
        className="movie-card__info"
      >
        <div
          className="movie-card__poster movie-card__poster--big"
        >
          <img
            alt="Seven Years in Tibet poster"
            height="327"
            src="https://es31-server.appspot.com/wtw/static/film/poster/Seven_Years_in_Tibet.jpg"
            width="218"
          />
        </div>
        <div
          className="movie-card__desc"
        >
          <nav
            className="movie-nav movie-card__nav"
          >
            <ul
              className="movie-nav__list"
            >
              <li
                className="movie-nav__item"
              >
                <a
                  className="movie-nav__link"
                  href="#"
                  onClick={[Function]}
                >
                  Overview
                </a>
              </li>
              <li
                className="movie-nav__item movie-nav__item--active"
              >
                <a
                  className="movie-nav__link"
                  href="#"
                  onClick={[Function]}
                >
                  Details
                </a>
              </li>
              <li
                className="movie-nav__item"
              >
                <a
                  className="movie-nav__link"
                  href="#"
                  onClick={[Function]}
                >
                  Reviews
                </a>
              </li>
            </ul>
          </nav>
          <div
            className="movie-card__text movie-card__row"
          >
            <div
              className="movie-card__text-col"
            >
              <p
                className="movie-card__details-item"
              >
                <strong
                  className="movie-card__details-name"
                >
                  Director
                </strong>
                <span
                  className="movie-card__details-value"
                >
                  Jean-Jacques Annaud
                </span>
              </p>
              <p
                className="movie-card__details-item"
              >
                <strong
                  className="movie-card__details-name"
                >
                  Starring
                </strong>
                <span
                  className="movie-card__details-value"
                >
                  Brad Pitt,
David Thewlis,
BD Wong
                </span>
              </p>
            </div>
            <div
              className="movie-card__text-col"
            >
              <p
                className="movie-card__details-item"
              >
                <strong
                  className="movie-card__details-name"
                >
                  Run Time
                </strong>
                <span
                  className="movie-card__details-value"
                >
                  2h 16m
                </span>
              </p>
              <p
                className="movie-card__details-item"
              >
                <strong
                  className="movie-card__details-name"
                >
                  Genre
                </strong>
                <span
                  className="movie-card__details-value"
                >
                  Adventure
                </span>
              </p>
              <p
                className="movie-card__details-item"
              >
                <strong
                  className="movie-card__details-name"
                >
                  Released
                </strong>
                <span
                  className="movie-card__details-value"
                >
                  1997
                </span>
              </p>
            </div>
          </div>
        </div>
      </div>
    </div>
  </section>
  <div
    className="page-content"
  >
    <section
      className="catalog catalog--like-this"
    >
      <h2
        className="catalog__title"
      >
        More like this
      </h2>
      <div
        className="catalog__movies-list"
      >
        <article
          className="small-movie-card catalog__movies-card"
          onClick={[Function]}
          onMouseEnter={[Function]}
          onMouseLeave={[Function]}
        >
          <a
            href="/films/1"
            onClick={[Function]}
          >
            <div
              className="small-movie-card__image"
            >
              <video
                height={175}
                muted={true}
                poster="https://es31-server.appspot.com/wtw/static/film/preview/seven-years-in-tibet.jpg"
                src="https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm"
                width={280}
              />
            </div>
          </a>
          <h3
            className="small-movie-card__title"
          >
            <a
              className="small-movie-card__link"
              href="/films/1"
              onClick={[Function]}
            >
              Seven Years in Tibet
            </a>
          </h3>
        </article>
        <article
          className="small-movie-card catalog__movies-card"
          onClick={[Function]}
          onMouseEnter={[Function]}
          onMouseLeave={[Function]}
        >
          <a
            href="/films/2"
            onClick={[Function]}
          >
            <div
              className="small-movie-card__image"
            >
              <video
                height={175}
                muted={true}
                poster="https://es31-server.appspot.com/wtw/static/film/preview/what-we-do-in-the-shadows.jpg"
                src="https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4"
                width={280}
              />
            </div>
          </a>
          <h3
            className="small-movie-card__title"
          >
            <a
              className="small-movie-card__link"
              href="/films/2"
              onClick={[Function]}
            >
              What We Do in the Shadows
            </a>
          </h3>
        </article>
        <article
          className="small-movie-card catalog__movies-card"
          onClick={[Function]}
          onMouseEnter={[Function]}
          onMouseLeave={[Function]}
        >
          <a
            href="/films/3"
            onClick={[Function]}
          >
            <div
              className="small-movie-card__image"
            >
              <video
                height={175}
                muted={true}
                poster="https://es31-server.appspot.com/wtw/static/film/preview/johnny-english.jpg"
                src="https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4"
                width={280}
              />
            </div>
          </a>
          <h3
            className="small-movie-card__title"
          >
            <a
              className="small-movie-card__link"
              href="/films/3"
              onClick={[Function]}
            >
              Johnny English
            </a>
          </h3>
        </article>
        <article
          className="small-movie-card catalog__movies-card"
          onClick={[Function]}
          onMouseEnter={[Function]}
          onMouseLeave={[Function]}
        >
          <a
            href="/films/4"
            onClick={[Function]}
          >
            <div
              className="small-movie-card__image"
            >
              <video
                height={175}
                muted={true}
                poster="https://es31-server.appspot.com/wtw/static/film/preview/bronson.jpg"
                src="https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm"
                width={280}
              />
            </div>
          </a>
          <h3
            className="small-movie-card__title"
          >
            <a
              className="small-movie-card__link"
              href="/films/4"
              onClick={[Function]}
            >
              Bronson
            </a>
          </h3>
        </article>
        <article
          className="small-movie-card catalog__movies-card"
          onClick={[Function]}
          onMouseEnter={[Function]}
          onMouseLeave={[Function]}
        >
          <a
            href="/films/5"
            onClick={[Function]}
          >
            <div
              className="small-movie-card__image"
            >
              <video
                height={175}
                muted={true}
                poster="https://es31-server.appspot.com/wtw/static/film/preview/macbeth.jpg"
                src="https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm"
                width={280}
              />
            </div>
          </a>
          <h3
            className="small-movie-card__title"
          >
            <a
              className="small-movie-card__link"
              href="/films/5"
              onClick={[Function]}
            >
              Macbeth
            </a>
          </h3>
        </article>
        <article
          className="small-movie-card catalog__movies-card"
          onClick={[Function]}
          onMouseEnter={[Function]}
          onMouseLeave={[Function]}
        >
          <a
            href="/films/6"
            onClick={[Function]}
          >
            <div
              className="small-movie-card__image"
            >
              <video
                height={175}
                muted={true}
                poster="https://es31-server.appspot.com/wtw/static/film/preview/seven-years-in-tibet.jpg"
                src="https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm"
                width={280}
              />
            </div>
          </a>
          <h3
            className="small-movie-card__title"
          >
            <a
              className="small-movie-card__link"
              href="/films/6"
              onClick={[Function]}
            >
              Seven Years in Tibet
            </a>
          </h3>
        </article>
        <article
          className="small-movie-card catalog__movies-card"
          onClick={[Function]}
          onMouseEnter={[Function]}
          onMouseLeave={[Function]}
        >
          <a
            href="/films/7"
            onClick={[Function]}
          >
            <div
              className="small-movie-card__image"
            >
              <video
                height={175}
                muted={true}
                poster="https://es31-server.appspot.com/wtw/static/film/preview/what-we-do-in-the-shadows.jpg"
                src="https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4"
                width={280}
              />
            </div>
          </a>
          <h3
            className="small-movie-card__title"
          >
            <a
              className="small-movie-card__link"
              href="/films/7"
              onClick={[Function]}
            >
              What We Do in the Shadows
            </a>
          </h3>
        </article>
        <article
          className="small-movie-card catalog__movies-card"
          onClick={[Function]}
          onMouseEnter={[Function]}
          onMouseLeave={[Function]}
        >
          <a
            href="/films/8"
            onClick={[Function]}
          >
            <div
              className="small-movie-card__image"
            >
              <video
                height={175}
                muted={true}
                poster="https://es31-server.appspot.com/wtw/static/film/preview/johnny-english.jpg"
                src="https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4"
                width={280}
              />
            </div>
          </a>
          <h3
            className="small-movie-card__title"
          >
            <a
              className="small-movie-card__link"
              href="/films/8"
              onClick={[Function]}
            >
              Johnny English
            </a>
          </h3>
        </article>
        <article
          className="small-movie-card catalog__movies-card"
          onClick={[Function]}
          onMouseEnter={[Function]}
          onMouseLeave={[Function]}
        >
          <a
            href="/films/9"
            onClick={[Function]}
          >
            <div
              className="small-movie-card__image"
            >
              <video
                height={175}
                muted={true}
                poster="https://es31-server.appspot.com/wtw/static/film/preview/bronson.jpg"
                src="https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm"
                width={280}
              />
            </div>
          </a>
          <h3
            className="small-movie-card__title"
          >
            <a
              className="small-movie-card__link"
              href="/films/9"
              onClick={[Function]}
            >
              Bronson
            </a>
          </h3>
        </article>
        <article
          className="small-movie-card catalog__movies-card"
          onClick={[Function]}
          onMouseEnter={[Function]}
          onMouseLeave={[Function]}
        >
          <a
            href="/films/10"
            onClick={[Function]}
          >
            <div
              className="small-movie-card__image"
            >
              <video
                height={175}
                muted={true}
                poster="https://es31-server.appspot.com/wtw/static/film/preview/macbeth.jpg"
                src="https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm"
                width={280}
              />
            </div>
          </a>
          <h3
            className="small-movie-card__title"
          >
            <a
              className="small-movie-card__link"
              href="/films/10"
              onClick={[Function]}
            >
              Macbeth
            </a>
          </h3>
        </article>
        <article
          className="small-movie-card catalog__movies-card"
          onClick={[Function]}
          onMouseEnter={[Function]}
          onMouseLeave={[Function]}
        >
          <a
            href="/films/11"
            onClick={[Function]}
          >
            <div
              className="small-movie-card__image"
            >
              <video
                height={175}
                muted={true}
                poster="https://es31-server.appspot.com/wtw/static/film/preview/seven-years-in-tibet.jpg"
                src="https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm"
                width={280}
              />
            </div>
          </a>
          <h3
            className="small-movie-card__title"
          >
            <a
              className="small-movie-card__link"
              href="/films/11"
              onClick={[Function]}
            >
              Seven Years in Tibet
            </a>
          </h3>
        </article>
        <article
          className="small-movie-card catalog__movies-card"
          onClick={[Function]}
          onMouseEnter={[Function]}
          onMouseLeave={[Function]}
        >
          <a
            href="/films/12"
            onClick={[Function]}
          >
            <div
              className="small-movie-card__image"
            >
              <video
                height={175}
                muted={true}
                poster="https://es31-server.appspot.com/wtw/static/film/preview/what-we-do-in-the-shadows.jpg"
                src="https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4"
                width={280}
              />
            </div>
          </a>
          <h3
            className="small-movie-card__title"
          >
            <a
              className="small-movie-card__link"
              href="/films/12"
              onClick={[Function]}
            >
              What We Do in the Shadows
            </a>
          </h3>
        </article>
        <article
          className="small-movie-card catalog__movies-card"
          onClick={[Function]}
          onMouseEnter={[Function]}
          onMouseLeave={[Function]}
        >
          <a
            href="/films/13"
            onClick={[Function]}
          >
            <div
              className="small-movie-card__image"
            >
              <video
                height={175}
                muted={true}
                poster="https://es31-server.appspot.com/wtw/static/film/preview/johnny-english.jpg"
                src="https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4"
                width={280}
              />
            </div>
          </a>
          <h3
            className="small-movie-card__title"
          >
            <a
              className="small-movie-card__link"
              href="/films/13"
              onClick={[Function]}
            >
              Johnny English
            </a>
          </h3>
        </article>
        <article
          className="small-movie-card catalog__movies-card"
          onClick={[Function]}
          onMouseEnter={[Function]}
          onMouseLeave={[Function]}
        >
          <a
            href="/films/14"
            onClick={[Function]}
          >
            <div
              className="small-movie-card__image"
            >
              <video
                height={175}
                muted={true}
                poster="https://es31-server.appspot.com/wtw/static/film/preview/bronson.jpg"
                src="https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm"
                width={280}
              />
            </div>
          </a>
          <h3
            className="small-movie-card__title"
          >
            <a
              className="small-movie-card__link"
              href="/films/14"
              onClick={[Function]}
            >
              Bronson
            </a>
          </h3>
        </article>
        <article
          className="small-movie-card catalog__movies-card"
          onClick={[Function]}
          onMouseEnter={[Function]}
          onMouseLeave={[Function]}
        >
          <a
            href="/films/15"
            onClick={[Function]}
          >
            <div
              className="small-movie-card__image"
            >
              <video
                height={175}
                muted={true}
                poster="https://es31-server.appspot.com/wtw/static/film/preview/macbeth.jpg"
                src="https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm"
                width={280}
              />
            </div>
          </a>
          <h3
            className="small-movie-card__title"
          >
            <a
              className="small-movie-card__link"
              href="/films/15"
              onClick={[Function]}
            >
              Macbeth
            </a>
          </h3>
        </article>
        <article
          className="small-movie-card catalog__movies-card"
          onClick={[Function]}
          onMouseEnter={[Function]}
          onMouseLeave={[Function]}
        >
          <a
            href="/films/16"
            onClick={[Function]}
          >
            <div
              className="small-movie-card__image"
            >
              <video
                height={175}
                muted={true}
                poster="https://es31-server.appspot.com/wtw/static/film/preview/seven-years-in-tibet.jpg"
                src="https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm"
                width={280}
              />
            </div>
          </a>
          <h3
            className="small-movie-card__title"
          >
            <a
              className="small-movie-card__link"
              href="/films/16"
              onClick={[Function]}
            >
              Seven Years in Tibet
            </a>
          </h3>
        </article>
        <article
          className="small-movie-card catalog__movies-card"
          onClick={[Function]}
          onMouseEnter={[Function]}
          onMouseLeave={[Function]}
        >
          <a
            href="/films/17"
            onClick={[Function]}
          >
            <div
              className="small-movie-card__image"
            >
              <video
                height={175}
                muted={true}
                poster="https://es31-server.appspot.com/wtw/static/film/preview/what-we-do-in-the-shadows.jpg"
                src="https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4"
                width={280}
              />
            </div>
          </a>
          <h3
            className="small-movie-card__title"
          >
            <a
              className="small-movie-card__link"
              href="/films/17"
              onClick={[Function]}
            >
              What We Do in the Shadows
            </a>
          </h3>
        </article>
        <article
          className="small-movie-card catalog__movies-card"
          onClick={[Function]}
          onMouseEnter={[Function]}
          onMouseLeave={[Function]}
        >
          <a
            href="/films/18"
            onClick={[Function]}
          >
            <div
              className="small-movie-card__image"
            >
              <video
                height={175}
                muted={true}
                poster="https://es31-server.appspot.com/wtw/static/film/preview/johnny-english.jpg"
                src="https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4"
                width={280}
              />
            </div>
          </a>
          <h3
            className="small-movie-card__title"
          >
            <a
              className="small-movie-card__link"
              href="/films/18"
              onClick={[Function]}
            >
              Johnny English
            </a>
          </h3>
        </article>
        <article
          className="small-movie-card catalog__movies-card"
          onClick={[Function]}
          onMouseEnter={[Function]}
          onMouseLeave={[Function]}
        >
          <a
            href="/films/19"
            onClick={[Function]}
          >
            <div
              className="small-movie-card__image"
            >
              <video
                height={175}
                muted={true}
                poster="https://es31-server.appspot.com/wtw/static/film/preview/bronson.jpg"
                src="https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm"
                width={280}
              />
            </div>
          </a>
          <h3
            className="small-movie-card__title"
          >
            <a
              className="small-movie-card__link"
              href="/films/19"
              onClick={[Function]}
            >
              Bronson
            </a>
          </h3>
        </article>
        <article
          className="small-movie-card catalog__movies-card"
          onClick={[Function]}
          onMouseEnter={[Function]}
          onMouseLeave={[Function]}
        >
          <a
            href="/films/20"
            onClick={[Function]}
          >
            <div
              className="small-movie-card__image"
            >
              <video
                height={175}
                muted={true}
                poster="https://es31-server.appspot.com/wtw/static/film/preview/macbeth.jpg"
                src="https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm"
                width={280}
              />
            </div>
          </a>
          <h3
            className="small-movie-card__title"
          >
            <a
              className="small-movie-card__link"
              href="/films/20"
              onClick={[Function]}
            >
              Macbeth
            </a>
          </h3>
        </article>
        <article
          className="small-movie-card catalog__movies-card"
          onClick={[Function]}
          onMouseEnter={[Function]}
          onMouseLeave={[Function]}
        >
          <a
            href="/films/21"
            onClick={[Function]}
          >
            <div
              className="small-movie-card__image"
            >
              <video
                height={175}
                muted={true}
                poster="https://es31-server.appspot.com/wtw/static/film/preview/seven-years-in-tibet.jpg"
                src="https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm"
                width={280}
              />
            </div>
          </a>
          <h3
            className="small-movie-card__title"
          >
            <a
              className="small-movie-card__link"
              href="/films/21"
              onClick={[Function]}
            >
              Seven Years in Tibet
            </a>
          </h3>
        </article>
        <article
          className="small-movie-card catalog__movies-card"
          onClick={[Function]}
          onMouseEnter={[Function]}
          onMouseLeave={[Function]}
        >
          <a
            href="/films/22"
            onClick={[Function]}
          >
            <div
              className="small-movie-card__image"
            >
              <video
                height={175}
                muted={true}
                poster="https://es31-server.appspot.com/wtw/static/film/preview/what-we-do-in-the-shadows.jpg"
                src="https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4"
                width={280}
              />
            </div>
          </a>
          <h3
            className="small-movie-card__title"
          >
            <a
              className="small-movie-card__link"
              href="/films/22"
              onClick={[Function]}
            >
              What We Do in the Shadows
            </a>
          </h3>
        </article>
        <article
          className="small-movie-card catalog__movies-card"
          onClick={[Function]}
          onMouseEnter={[Function]}
          onMouseLeave={[Function]}
        >
          <a
            href="/films/23"
            onClick={[Function]}
          >
            <div
              className="small-movie-card__image"
            >
              <video
                height={175}
                muted={true}
                poster="https://es31-server.appspot.com/wtw/static/film/preview/johnny-english.jpg"
                src="https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4"
                width={280}
              />
            </div>
          </a>
          <h3
            className="small-movie-card__title"
          >
            <a
              className="small-movie-card__link"
              href="/films/23"
              onClick={[Function]}
            >
              Johnny English
            </a>
          </h3>
        </article>
        <article
          className="small-movie-card catalog__movies-card"
          onClick={[Function]}
          onMouseEnter={[Function]}
          onMouseLeave={[Function]}
        >
          <a
            href="/films/24"
            onClick={[Function]}
          >
            <div
              className="small-movie-card__image"
            >
              <video
                height={175}
                muted={true}
                poster="https://es31-server.appspot.com/wtw/static/film/preview/bronson.jpg"
                src="https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm"
                width={280}
              />
            </div>
          </a>
          <h3
            className="small-movie-card__title"
          >
            <a
              className="small-movie-card__link"
              href="/films/24"
              onClick={[Function]}
            >
              Bronson
            </a>
          </h3>
        </article>
        <article
          className="small-movie-card catalog__movies-card"
          onClick={[Function]}
          onMouseEnter={[Function]}
          onMouseLeave={[Function]}
        >
          <a
            href="/films/25"
            onClick={[Function]}
          >
            <div
              className="small-movie-card__image"
            >
              <video
                height={175}
                muted={true}
                poster="https://es31-server.appspot.com/wtw/static/film/preview/macbeth.jpg"
                src="https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm"
                width={280}
              />
            </div>
          </a>
          <h3
            className="small-movie-card__title"
          >
            <a
              className="small-movie-card__link"
              href="/films/25"
              onClick={[Function]}
            >
              Macbeth
            </a>
          </h3>
        </article>
      </div>
    </section>
    <footer
      className="page-footer"
    >
      <div
        className="logo"
      >
        <a
          className="logo__link logo__link--light"
          href="/"
        >
          <span
            className="logo__letter logo__letter--1"
          >
            W
          </span>
          <span
            className="logo__letter logo__letter--2"
          >
            T
          </span>
          <span
            className="logo__letter logo__letter--3"
          >
            W
          </span>
        </a>
      </div>
      <div
        className="copyright"
      >
        <p>
          © 2019 What to watch Ltd.
        </p>
      </div>
    </footer>
  </div>
</div>
`;
